content/articles/methods/ltca.md

```diff
@@ -0,0 +1,365 @@
+</p>
+
+(3) Delayed reward phase: Agent should move to a green goal grid cell as quickly as possible.  If the agent is carrying the key when it reaches the goal, it is rewarded extra points.
+
+<p align="center">
+<img src="images/ltca_gridworld_p3.png" alt="Phase 1" style="width:190px;">
+</p>
+
+The agent remains in each phase for a fixed period of time, regardless of how quickly it finishes the intermediate task, and then teleports to the next phase.  The environment resets with a different random seed that randomizes the placement of the key in phase 1 and distractor objects in phase 2.
+
+Note, unlike the Reconstructive Memory Agent in [1], our agent does not have the ability to encode and reconstruct memories, and our environment is not set up to test for that ability.
+
+## Experiments
+
+In all experiments, the time spent in phase 1 and phase 3 is fixed, as well as the distal reward in phase 3: 5 points for reaching the goal without a key and 20 points for reaching the goal carrying a key (with a small penalty proportional to step count to encourage moving quickly to the goal).
+
+Each experiment varies a particular parameter in the second phase to evaluate the effect on learning for the baseline A2C algorithm compared to A2C enhanced with TRT.
+
+The focus our evaluation for each experiment is on the last reward (the distal reward) obtained in phase 3, although we also show the overall returns to make sure the agent is also learning a policy that also maximizes the rewards in phase 2.
+
+
+### Time delay in distractor phase
+
+We vary the time spent in the distractor phase, $T_2$, as a multiple of the discount factor timescale.  We used a discount factor of $\gamma=0.99$, which corresponds to a timescale of ~100 steps according to (\ref{discount-timescale}).  We ran experiments for $T_2 = (0, 0.5, 1, 2) * \tau_{\gamma}$.
+
+<p align="center">
+<img src="images/ltca_time_delay_expt_plots.png" alt="P2 time delay expt" style="width:800px;">
+</p>
+
+<small>Fig 1. Returns in phase 3 for time delays in phase 2 of 0.5$\tau_{\gamma}$, $\tau_{\gamma}$, and 2$\tau_{\gamma}$.
+</small>
+
+### Distractor reward size
+
+We vary the size of the distractor rewards, 4 gifts for the agent to toggle open, in phase 2.  We run experiments for a reward of 0, 1, 5, and 8, resulting in maximum possible rewards in phase 2 of 0, 4, 20, and 32.
+
+In comparison, the maximum possible reward in phase 3 is 20.
+
+<p align="center">
+<img src="images/ltca_reward_expt_plots.png" alt="P2 reward size expt" style="width:800px;">
+</p>
+
+<small>Fig 2. Returns in phase 3 for rewards in phase 2 of size 0, 5, and 8.
+</small>
+
+
+### Distractor reward variance
+
+We fix the mean reward size of the gifts in phase 2 at 5, but the variance of the rewards ranges from 0, to 1.33, to 8.33, corresponding to drawing each reward from a uniform distribution centered at 5, with minimum and maximum ranges of [5, 5], [3, 7], and [0, 10], respectively.
```

Comment:
I'm not sure what we get out of the variances ... the ranges are more useful.

Reply:
reworded